packages/client/src/object/aggregate.test.ts

```diff
@@ -171,13 +186,7 @@ describe("aggregate", () => {
       >
     >(false); // subSelect should hide unused keys
 
-    const grouped = await aggregate(
-      clientCtx,
-      objectTypeWithAllPropertyTypes,
-      {
-        type: "base",
-        objectType: "ToDo",
-      },
+    const grouped = await client(objectTypeWithAllPropertyTypes).aggregate(
```

Comment:
For some reason type tests were having issues when using the aggregate function directly, but were all good when using the user exposed aggregate function. This should be fine since the user facing stuff works, but just flagging

Reply:
For some reason, the aggregate function was not capturing clause types correctly and everything was still in generic form (e.g. like it wasn't getting the concrete information passed in `$select: {myProp:asc}`